cmd/kube-proxy/app/server.go

```diff
@@ -461,8 +461,9 @@ with the apiserver API to configure the proxy.`,
 
 	opts.AddFlags(cmd.Flags())
 
-	// TODO handle error
-	cmd.MarkFlagFilename("config", "yaml", "yml", "json")
+	if err := cmd.MarkFlagFilename("config", "yaml", "yml", "json"); err != nil {
+		klog.Fatalf("unable to mark flag filename: %v", err)
```

Comment:
No, that was just a silly comment. No other use of `cmd.MarkFlagFilename` in kubernetes checks the error. Just put it back how it was and remove the comment.